pkg/kubelet/cm/deviceplugin/manager_stub.go

```diff
@@ -23,30 +23,35 @@ import (
 	"k8s.io/kubernetes/plugin/pkg/scheduler/schedulercache"
 )
 
-// HandlerStub provides a simple stub implementation for Handler.
-type HandlerStub struct{}
+// ManagerStub provides a simple stub implementation for Handler.
```

Comment:
s/for Handler/for Manager

Reply:
Gotcha - thanks! Did a scan for 'handler'.
We do have some old mentions of the ListWatch on the endpoint, we should change as well.